src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -417,18 +420,13 @@ public void getColumns() throws SQLException {
         assertThat(rs.next()).isFalse();
 
         rs = meta.getColumns(null, null, "%", "%");
-        // SYSTEM TABLE "sqlite_schema"
-        assertThat(rs.next()).isTrue();
-        assertThat(rs.getString("TABLE_NAME")).isEqualTo("sqlite_schema");
-        assertThat(rs.getString("COLUMN_NAME")).isEqualTo("type");
-        assertThat(rs.next()).isTrue();
-        assertThat(rs.getString("COLUMN_NAME")).isEqualTo("name");
-        assertThat(rs.next()).isTrue();
-        assertThat(rs.getString("COLUMN_NAME")).isEqualTo("tbl_name");
-        assertThat(rs.next()).isTrue();
-        assertThat(rs.getString("COLUMN_NAME")).isEqualTo("rootpage");
-        assertThat(rs.next()).isTrue();
-        assertThat(rs.getString("COLUMN_NAME")).isEqualTo("sql");
+
+        // SYSTEM TABLE "sqlite_schema" for main
+        assertSystemSchema(rs);
+
+        // SYSTEM TABLE "sqlite_schema" for temp
```

Comment:
where is that `temp` database attached? i can't find any mention to it

Reply:
temp is the sqlite temporary files, I think are returned as system external schema if any exist.